src/nunavut/lang/c/support/nunavut_common.h

```diff
@@ -0,0 +1,389 @@
+/*
+ *
+ * UAVCAN common serialization support routines.                                                                     +-+ +-+
+ * 
+ * Not meant to be included directly by a user. Please include nunavut.h
+ * or nunavut_le.h instead, depending on your platform architecture.
```

Comment:
One problem with this way of doing things is that `nunavut.h` and `nunavut_le.h` do not have all of the function declarations directly in them and thus do not serve nicely as an API reference.

The solutions all seem kind of gross to me:
- Put all declarations above `#include "nunavut_common.h"` in both LE and general.
- Put all declarations at top of `nunavut_common.h`
- Leave it as is.

Reply:
The ideal would be to only generate the version you were going to use (i.e. no dead code). If I provided a way to do this would you agree this is the right approach?

Reply:
opened #123 to provide this for you. ETA, Monday?

Reply:
The change is up for review: https://github.com/UAVCAN/nunavut/pull/125

Reply:
The change is in. #125 is available to you now.